src/components/organisms/sections/ContactForm/index.jsx

```diff
@@ -70,6 +84,9 @@ class ContactForm extends React.Component {
   handleSubmit = (e) => {
     e.preventDefault();
 
+    // Analytics
+    this.props.googleAnalytics.registerContactSend();
+
```

Comment:
It this newline wanted? In my opinion there is no reason to separate those two function calls.